packages/neos-ui/src/Containers/LeftSideBar/NodeTree/index.js

```diff
@@ -123,6 +146,13 @@ export default class NodeTree extends PureComponent {
 
         return (
             <Tree className={classNames}>
+                <a
+                    role="button"
+                    onClick={this.handleCollapseAll}
+                    className={style.collapseAll}
+                >
+                    Collapse All
```

Comment:
I think a style similar to how PHPStorm does it in the dark mode would fit better:
![CleanShot 2024-03-19 at 10 58 40@2x](https://github.com/neos/neos-ui/assets/596967/3751df5b-ce12-46b3-97d6-8f17211cd1c9)

Reply:
Heya, the closest Icon I could find was "compress-alt". For clarity reasons, I made the Icon a little bigger and added a "Collapse All" title. I like how it looks now , but I am open to other suggestions aswell.

![image](https://github.com/neos/neos-ui/assets/56877180/347c9479-8897-4b48-9b46-4904871c5714)

Reply:
Awesome!